src/screens/leave/LeaveDetails.tsx

```diff
@@ -230,7 +229,14 @@ class LeaveDetails extends React.Component<
                     color: theme.palette.secondary,
                     paddingBottom: theme.spacing,
                   }}>
-                  {leaveDates}
+                  <Date nested>{employeeLeaveRequest?.fromDate}</Date>
```

Comment:
Refactor "Date" component to a specific name.

Reply:
Resolved